test/unittests/test_intx_api.cpp

```diff
@@ -259,35 +259,52 @@ TYPED_TEST(uint_api, comparison)
     EXPECT_TRUE(d >= x);
 }
 
+struct W
+{
+    intx::uint256 v;
+
+    friend auto operator<=>(W, W) = default;
+};
+
+struct W128
+{
+    intx::uint128 v;
+    friend auto operator<=>(W128, W128) = default;
+};
+
 TYPED_TEST(uint_api, spaceship_operator)
 {
-    auto a = uint256{1};
```

Comment:
Let's keep the old test too.

Reply:
I returned the old test back, and thought it is better to keep it seprate with the newly created test. if it's better to have them in one, please let me know.